### ISSUE TITLE
t.co links remain as twitter containers after the redirect

### ISSUE THREAD

**Issue by aventuraviper** [v1shwa/contain-twitter#2]
Hi,

I've noticed that clicking a link in a tweet that is automatically changed to a _t.co_ URL by twitter (which then redirects to the actual link in the tweet) results in the original link being opened incorrectly in a twitter container.

For example, the link in this tweet from firefox: [https://twitter.com/firefox/status/1237337410071080962](https://twitter.com/firefox/status/1237337410071080962)

That link is automatically converted to: [https://t.co/ikX7pfVxzA?amp=1](https://t.co/ikX7pfVxzA?amp=1) within the tweet so clicking it results in opening within a twitter container. However, upon redirection to the original link ([https://support.mozilla.org/en-US/kb/firefox-uses-too-much-memory-or-cpu-resources](https://support.mozilla.org/en-US/kb/firefox-uses-too-much-memory-or-cpu-resources)) the twitter container is not closed with the link opened in no container.

The same actually occurs for any link in a tweet, but is far less noticeable in the majority as they liks in the tweets are themselves url shorteners such is _bit.ly_ or _go.nasa.gov_ etc. Because of the second redirect the end URL is correctly opened in no container and the twitter container closed.

Thanks

**Comment by v1shwa** [comment]
Thanks for reporting the issue @aventuraviper .

I would be glad to merge your PR if you can submit a fix. :) Otherwise, I will try to fix it over the next couple of days. 

Thanks,
Vishwa

**Comment by v1shwa** [comment]
@aventuraviper This only seems to be happening with mozilla.com links. All other links (including the ones that are not shortend already) seems to redirecting to no container.

Let me know if you have noticed any non mozilla links that are not redirecting.

**Comment by aventuraviper** [comment]
@v1shwa Sorry for very delayed response.

After further use I concur that it is only mozzilla.com. I actually noticed the same problem for firefox AMO links (mozilla.org) with a reddit container extension (https://github.com/SleepingPanda/contain-reddit).

That is also a fork of facebook container so presumably there is something in that underlying addon that treats mozilla links differently?

**Comment by v1shwa** [comment]
@aventuraviper 
Yeah, I guess it's a Firefox level restriction as I don't see anything in the addon code that does any special treatment for the mozilla.org links.

Anyway, I have raised this issue with the original facebook-container repo here. https://github.com/mozilla/contain-facebook/issues/622  

Let's see what they say

**Comment by grahamperrin** [comment]
I have the symptom, not limited to Mozilla links. Maybe the result of my choice of extensions.
